net.sonic_pi.SonicPi.json

```diff
@@ -0,0 +1,230 @@
+{
+    "id": "net.sonic_pi.SonicPi",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.15",
+    "runtime": "org.kde.Platform",
+    "sdk": "org.kde.Sdk",
+    "runtime-version": "5.15",
+    "command": "sonic-pi",
+    "rename-icon": "sonic-pi",
+    "rename-desktop-file": "sonic-pi.desktop",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc",
+        "--socket=x11",
```

Comment:
Should be `fallback-x11` if it works natively on Wayland.

Reply:
I actually now disabled Wayland, it works better, like a lot of Qt apps.